app/routes/docs.$lang.$ref.tsx

```diff
@@ -334,7 +334,9 @@ function VersionLink({
   }
 
   return to ? (
-    <Link to={to} className={className}>
+    // No need to discover these versioned links since they all match the
+    // current route we're on but with different params
+    <Link to={to} className={className} discover="none">
```

Comment:
Don't bother discovering the versioned links in the header dropdown since they all link to the current route